kili/queries/label/__init__.py

```diff
@@ -300,21 +241,15 @@ def export_labels_as_df(self,
         """
         projects = QueriesProject(self.auth).projects(project_id)
         assert len(projects) == 1, NO_ACCESS_RIGHT
-        project = projects[0]
-        if 'interfaceCategory' not in project:
-            return pd.DataFrame()
-
-        interface_category = project['interfaceCategory']
         assets = QueriesAsset(self.auth).assets(
-            project_id=project_id, fields=asset_fields + ['labels.' + field for field in fields])
+            project_id=project_id,
+            fields=asset_fields + ['labels.' + field for field in fields])
         labels = [dict(label, **dict((f'asset_{key}', asset[key]) for key in asset if key != 'labels'))
                   for asset in assets for label in asset['labels']]
         labels_df = pd.DataFrame(labels)
-        if 'jsonResponse' in labels_df.columns:
-            labels_df['jsonResponse'] = labels_df['jsonResponse'].apply(
-                lambda json_response: QueriesLabel.parse_json_response(json_response, interface_category))
```

Comment:
Don't remove those lines, instead change `parse_json_response` so that it does not need `interface_category`